tests/virtual_machines_test.go

```diff
@@ -497,8 +497,7 @@ var _ = Describe("Virtual Machines", func() {
 				_, err = net.ParseMAC(vm.Spec.Template.Spec.Domain.Devices.Interfaces[0].MacAddress)
 				Expect(err).ToNot(HaveOccurred())
 
-				By("deleting leader manager")
-				DeleteLeaderManager()
+				ChangeManagerLeadership()
```

Comment:
Please add that line with 'By', in case something goes wrong in tests suite we will know 
where it comes from

Reply:
Added inside `ChangeManagerLeadership`